src/backend/tools/index.ts

```diff
@@ -191,6 +191,10 @@ export const DXVK = {
       return true
     }
 
+    if (isMac && tool !== 'dxvk') {
+      return true
+    }
```

Comment:
Somewhat of the same issue here: Instead of just returning true for macOS, change callers of this so they don't try to do this when on macOS

Reply:
the thing here is that we don't handle this logic on frontend, that is the one that calls this method.
We only ask for dxvk and then the installer figure it out what tool to install.

Reply:
So what I gather from that is that the Frontend should only ever request the `dxvk` tool... So why do you need this check then?

Anyways, assuming you've tested this for all Wine types available on macOS, I don't see a huge issue here.
We should go through and actually figure out what's wrong here, but that can happen someday in the future